### ISSUE TITLE
Multi-View如何预测一个mesh出来？(How to get one single mesh for multi-view hrn)

### ISSUE THREAD

**Issue by yxwktdk** [youngLBW/HRN#70]
根据之前的issue，您提出可以将不同视角的displacement和coeff进行平均以得到最后的整体的mesh，请问可否请您详细说明一下，该如何进行平均，以及平均的具体是哪几个值呢？比如针对下面的代码：【Based on the previous issue, you suggested that we could average the displacement and coefficient from different perspectives to obtain the final overall mesh. Could you please explain in detail how to perform this averaging, and which specific values are to be averaged? For example, in the following code:】
```    def save_results_mvhrn(self, out_dir, save_name='test'):
        results_list = []
        for i in range(self.n_views):
            # change value for computing visualization results
            self.input_img = self.input_img_list[i]
            self.pred_face_mid = self.pred_face_mid_list[i]
            self.pred_mask = self.pred_mask_list[i]
            self.pred_face_high = self.pred_face_high_list[i]
            self.deformation_map = self.canonical_deformation_map
            self.displacement_map = self.displacement_map_list[i]
            self.face_albedo_map = self.face_albedo_map_list[i]
            self.de_retouched_albedo_map = self.de_retouched_albedo_map_list[i]
            self.tex_valid_mask = self.tex_valid_mask_list[i]
            self.pred_vertex = self.pred_vertex_list[i]
            self.pred_color_high = self.pred_color_high_list[i]

            self.extra_results = self.extra_results_list[i]

            results = self.save_results(out_dir, '{}_view_{}'.format(save_name, i+1))
            results_list.append(results)


        return results_list```
是要将`self.displacement_map = self.displacement_map_list[i]`变为`self.displacement_map = torch.mean(self.displacement_map_list[i], dim=0)`吗？期待您的答复！
